### ISSUE TITLE
exposure to silica dust

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10285]
**Preferred term label:**
exposure to silica dust

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C2220359

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Silicosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000192
- label: Silica dust exposure
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: PMID:22529751
- def: History of exposure to respirable crystalline silica (SiO2).
- comment: Silicosis is a well known consequence of silica dust exposure, and exposure has also been associated with the risk of lung cancer, pulmonary tuberculosis, and other airway diseases.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10285
- synonyms:
